pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -3,10 +3,12 @@
 , glibmm, libsigcxx, lcms, boost, gettext, makeWrapper
 , gsl, python2, poppler, imagemagick, libwpg, librevenge
 , libvisio, libcdr, libexif, potrace, cmake, hicolor-icon-theme
+, scourSupport ? false
```

Comment:
Does this really need to be `false`?
It shouldn't affect the closure too much.

Reply:
I changed that to `false` as @FRidh queried it in the initial PR. But I can change it back to `true`.

Reply:
I haven't checked the closure size. How is Inkscape with optional dependencies? Are there a lot?

Reply:
Arch linux seems to have a list of [them](https://www.archlinux.org/packages/extra/x86_64/inkscape/).

Reply:
scour is the only optional dependency and it depends on `six`. I'll see if I can find some docs on checking the closure size.

Reply:
@FRidh, @worldofpeace this is the closure of scour, it looks small enough to me:

```
/nix/store/g2yk54hifqlsjiha3szr4q3ccmdzyrdv-glibc-2.27                 	   25010568
/nix/store/pmvnf35kg6k2p0z91ac663akazlf7wz6-bzip2-1.0.6.0.1            	   25083424
/nix/store/1vb4m694aj9bdm7aq3911yjjb73gn4ii-attr-2.4.47                	   25090384
/nix/store/di0d17idlw6fxf3qwpfyxciw94ywi664-zlib-1.2.11                	   25135080
/nix/store/f6a7rladm0r8k62qvr4yb3gyj81cr9yp-acl-2.2.52                 	   25190248
/nix/store/9l9mnc25y5rw4m6jfbgihv610si6gsiv-gdbm-1.18                  	   25644808
/nix/store/r47p5pzx52m3n34vdgqpk5rvqgm0m24m-bash-4.4-p23               	   26186104
/nix/store/ck5ay23hsmlc67pg3m34kzd1k2hhvww0-sqlite-3.24.0              	   26294200
/nix/store/dzvbhkzfgwkq97ljkcvi4gs7vmflh6a0-coreutils-8.30             	   26877336
/nix/store/iirmif7qgp7pgbv80z5x1sj3hbay893d-ncurses-6.1                	   28370672
/nix/store/ra2kcrzvp2hmsvmq82kma077h19yx0hr-openssl-1.0.2p             	   28686952
/nix/store/nhfaids6kfq0x6xrff9bl5qkyg83vvmf-readline-6.3p08            	   28758392
/nix/store/3v5r7fkrbkw2qajadvjbf6p6qriz9p1i-gcc-7.3.0-lib              	   30635424
/nix/store/i209v6zy67lpyybx6kxkrlvz6crv8dkp-db-5.3.28                  	   34871512
/nix/store/2brlr94ahy3a9mvcjy0qbqpv8zrb7b7s-python-2.7.15              	   94103328
/nix/store/cx4364dn6qx4xci990q1bk6i5z4880kw-python2.7-setuptools-40.2.0	   95128600
/nix/store/281q69f2wkzi9bdwfs2rlalcyrhrd2yb-python2.7-six-1.11.0       	   95196656
/nix/store/8baq3rijziz6ws95jih09la954achxvl-python2.7-scour-0.37       	   95514616
```